### DISCUSSION TITLE
Hacked account

### DISCUSSION THREAD

**Discussion by ghost** [community/community#132068] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

My account has been hacked, email : chadyachraf@outlook.com 
Please don't close this thread until my problem is resolved. I contacted Github support **2 MONTHS** ago !!!! _**AND STILL NO REPLY**_ this is absurd.
Please forward my request to the support. I need to regain access to my account.
Username : Chady00
Email : chadyachraf@outlook.com ( this email has been altered by the hacker / removed ) and probably my account is suspended.
Please help me restore it and don't end discussion or close thread until my problem is resolved. forward this to github please my projects are all lost.
I have all evidence that this is my account. Dont MARK THIS QUESTION AS SOLVED NOR END THE DISCUSSION BECAUSE MY PROBLEM IS NOT SOLVED YET.
FORWARD MY MESSAGE TO GITHUB I NEED TO RESTORE MY ACCOUNT AND I HAVE ALL PROOF WHAT'S TAKING SO LONG it's been 2 months!!!!!!!!!!!!!!!!!!!

**Comment by bacngv** [comment]
Oh

**Comment by catunderrain** [comment]
Good luck bro
